instrumentation/restlet/restlet-1.0/library/src/test/groovy/io/opententelemetry/instrumentation/restlet/v1_0/RestletServerTest.groovy

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opententelemetry.instrumentation.restlet.v1_0
+
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import io.opentelemetry.instrumentation.restlet.v1_0.AbstractRestletServerTest
+import io.opentelemetry.instrumentation.restlet.v1_0.RestletTracing
+import io.opentelemetry.instrumentation.test.LibraryTestTrait
+import org.restlet.Restlet
+
+class RestletServerTest extends AbstractRestletServerTest implements LibraryTestTrait {
+
+  @Override
+  Restlet wrapRestlet(Restlet restlet, String path){
+    RestletTracing tracing = RestletTracing.create(openTelemetry)
+    def filter = tracing.newFilter(path)
+    filter.setNext(restlet)
+    return filter
+  }
+
+  @Override
+  boolean testException() {
+    false //Filter's afterHandle does not execute if exception was thrown in the next restlet
```

Comment:
if `afterHandle` is not executed, will the Scope that was started in `beforeHandle` be left open (and the Context left bound to the thread)?

Reply:
after some investigation and looking into Restlet's `StatusFilter` I found a way to make `afterHandle` execute even if an exception was thrown, so the Scope will be safely closed in every case now